redbot/cogs/warnings/warnings.py

```diff
@@ -340,33 +340,57 @@ def __init__(self, bot: Red):
             pass
         await ctx.send(_("User {user} has been warned.").format(user=user))
 
-    @commands.command()
+    @commands.command(usage="")
     @commands.guild_only()
     async def warnings(
         self, ctx: commands.Context, user: Optional[Union[discord.Member, int]] = None
     ):
-        """List the warnings for the specified user.
-
-        Omit `<user>` to see your own warnings.
-
-        Note that showing warnings for users other than yourself requires
-        appropriate permissions.
-        """
+        """Check your own warnings."""
         if user is None:
             user = ctx.author
-        else:
+        else:  # TODO: remove this else statement in a release after 3.3.0 (will allow users time start using [p]warncheck)
             if not await is_admin_or_superior(self.bot, ctx.author):
                 return await ctx.send(
                     warning(_("You are not allowed to check warnings for other users!"))
                 )
+            else:
+                return await ctx.send(
+                    _("Checking other user's warnings has moved to `{prefix}warncheck`.").format(
+                        prefix=ctx.prefix
+                    )
+                )
 
-            try:
-                userid: int = user.id
-            except AttributeError:
-                userid: int = user
-                user = ctx.guild.get_member(userid)
-                user = user or namedtuple("Member", "id guild")(userid, ctx.guild)
```

Comment:
This code is unreachable, if user is none,  instead of doing a if/else check you could shorter is with a simpler “If user is not None” and as such there would be no  no need of the else